src/components/interview.ts

```diff
@@ -29,9 +33,9 @@ export const getInterviewer = async (id: string): Promise<Interviewer | undefine
   return await db.get('SELECT * FROM interviewers WHERE user_id = ?', id);
 };
 
-export const getDomainsString = (domains: string[]): string => _.join(domains, ', ');
+export const getDomainsString = (domains: string[]): string => '`' + _.join(domains, '`, `') + '`';
```

Comment:
nit: use template string

Reply:
is it actually better in this case? I have to escape the backticks which make it look really awkward.

Reply:
I think in general we want to use template strings as much as possible. In this case the backticks made it uglier than normal but we should try to be consistent.